### DISCUSSION TITLE
React Native better support

### DISCUSSION THREAD

**Discussion by Stevemoretz** [permafrost-dev/node-ray#47] [Ideas]
Hello, nice package. 
using : 

```js
import {ray} from "node-ray/web";
```
already works, however some stuff don't work, such as `console` object not turning into ray automatically, also the stack is not very good, and source mapping is not probably used.

```
17:12:01
ss255225
AppEntry.bundle?platform=ios&dev=true&hot=false&minify=false:419345
```

Using react native expo sdk 42, you can easily install and test.

**Comment by patinthehat** [comment]
> also the stack is not very good

Sorry for the late reply - please elaborate, I'm not clear on what you mean.  Please also list anything else that you feel can be improved.  Thanks! :+1:

**Comment by patinthehat** [comment]
Closing this discussion due to lack of response/activity. Please reopen a discussion or issue if you 
 you are still experiencing an issue and can provide more context and examples.
